### ISSUE TITLE
Stacked BarChartView: Position of Value-Labels

### ISSUE THREAD

**Issue by AndreasIgelCC** [danielgindi/ios-charts#777]
Is there any option to set the Position of Value-Labels in Stacked BarChartViews? Is this feature in focus or is it not planed to be implemented?
I have something like that in Mind (Possibility to rotate Value-Labels and to position them inside the Stack):
<img width="710" alt="screenshot_279" src="https://cloud.githubusercontent.com/assets/2921976/13349839/97516e54-dc7c-11e5-8bd9-29ae015086fb.png">

**Comment by liuxuan30** [comment]
it can show each sub bar's label already, but you cannot change the position with a simple enum.
What you can do is looking at bar chart renderer's drawValues() to change the position. A PR is more than welcome if you think it's a good feature to have and implemented in a flexible way.

**Comment by MiladFaridnia** [comment]
After a year it's not implemented as a feature yet ?

**Comment by liuxuan30** [comment]
yep, lack of time + people working on new features; even reviews, plus android sync..
